[2023-03-16T18:54:35.040+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 0_Employee_Sync_Granular_Steps.save_user_data manual__2023-03-16T18:54:29.497181+00:00 [queued]>
[2023-03-16T18:54:35.047+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 0_Employee_Sync_Granular_Steps.save_user_data manual__2023-03-16T18:54:29.497181+00:00 [queued]>
[2023-03-16T18:54:35.048+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-03-16T18:54:35.048+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-03-16T18:54:35.049+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-03-16T18:54:35.056+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): save_user_data> on 2023-03-16 18:54:29.497181+00:00
[2023-03-16T18:54:35.062+0000] {standard_task_runner.py:55} INFO - Started process 45779 to run task
[2023-03-16T18:54:35.066+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '0_Employee_Sync_Granular_Steps', 'save_user_data', 'manual__2023-03-16T18:54:29.497181+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_granular.py', '--cfg-path', '/tmp/tmprw9dpzkm']
[2023-03-16T18:54:35.068+0000] {standard_task_runner.py:83} INFO - Job 72: Subtask save_user_data
[2023-03-16T18:54:35.117+0000] {task_command.py:376} INFO - Running <TaskInstance: 0_Employee_Sync_Granular_Steps.save_user_data manual__2023-03-16T18:54:29.497181+00:00 [running]> on host 9fe6516b8a01
[2023-03-16T18:54:35.161+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=0_Employee_Sync_Granular_Steps
AIRFLOW_CTX_TASK_ID=save_user_data
AIRFLOW_CTX_EXECUTION_DATE=2023-03-16T18:54:29.497181+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-16T18:54:29.497181+00:00
[2023-03-16T18:54:35.174+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-16T18:54:35.183+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=0_Employee_Sync_Granular_Steps, task_id=save_user_data, execution_date=20230316T185429, start_date=20230316T185435, end_date=20230316T185435
[2023-03-16T18:54:35.238+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-03-16T18:54:35.260+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
